source/core.py

```diff
@@ -251,6 +251,54 @@ def onEndSession(evt):
 	import windowUtils
 	class MessageWindow(windowUtils.CustomWindow):
 		className = u"wxWindowClassNR"
+		oldBatteryStatus = None
+		hasBeenInitialized = False

```

Comment:
I still wonder whether we can get rid of this variable and instead just have an argument (onlyCache=True or similar) for handlePowerStatusChange which only gets passed in the constructor.